src/pages/components/ProductList/ProductList.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import "./ProductList.scss";
+
+class ProductList extends Component {
+  state = { ProductList: "" };
+
+  componentDidMount() {
+    fetch("http://localhost:3000/data/list.json", {
+      method: "GET"
+    })
```

Comment:
```suggestion
    fetch("/data/list.json")
```
- 로컬에 있는 폴더에 접근할 경우 위와 같이 주소를 짧게 써줄 수 있습니다
- get 요청은 두번째 인자를 생략해줄 수 있습니다

Reply:
오! 수정했습니다!